app.js

```diff
@@ -28,8 +31,12 @@ const config = {
           message: INVALID_PERMISSIONS,
         };
       }
+      const userData = await userService.getUserByFieldOrThrow(
```

Comment:
Why have you made this change? It doesn't affect anything except introducing another variable

Reply:
solved